database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -21,6 +21,7 @@ public function up()
             $table->string('district')->nullable();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->string('phone_number')->nullable();
```

Comment:
Hey @aaraamuthan Next time, don't edit the existing migrations. Because, this will require us to do `php artisan migrate:fresh`, which will delete all of our data. We donot want that. So next time, create a new migration called "edit_user_profile_add_phone_number" or something similar. 

I'm approving it this time.